client/src/components/Navbar.jsx

```diff
@@ -1,35 +1,179 @@
-import { AppBar, Box, Link, Stack } from "@mui/material";
-import React, { useEffect } from "react";
+import React, { useContext, useState } from "react";
+import {
+  AppBar,
+  Box,
+  Grid,
+  Link,
+  Typography,
+  MenuItem,
+  Popover,
+  IconButton,
+  Button,
+} from "@mui/material";
+import MenuIcon from "@mui/icons-material/Menu";
+import TrackerIcon from "../assets/tracker_icon.svg";
+import AccountIcon from "../assets/account_icon.svg";
+import LogoPath from "../assets/internship_tracker_logo.svg";
+import { AuthContext } from "../App";
 
-import bobaTalksLogoPath from "../assets/bobatalks_wide.svg";
+const pageNames = ["Find an internship", "About"];
+const pageLinks = ["/search", "/about"];
 
 const NavBar = () => {
-  useEffect(() => {
-    const image = new Image();
-    image.src = bobaTalksLogoPath;
-  }, []);
+  const [anchorElNav, setAnchorElNav] = useState(null);
+  const [authUser] = useContext(AuthContext);
 
+  const handleOpenNavMenu = (event) => {
+    setAnchorElNav(event.currentTarget);
+  };
+
+  const handleCloseNavMenu = () => {
+    setAnchorElNav(null);
+  };
```

Comment:
https://mui.com/material-ui/react-drawer/

MUI provides a lot of built in functionality so we don't need to create and track our own state for functionality like this

Reply:
![image](https://user-images.githubusercontent.com/12107969/235411398-3fe58e5a-1b57-415c-ad3e-2afdc1890a76.png)
Also the hover on the hamburger icon is skewed